typescript-demo/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM cgr.dev/chainguard/node AS base
+
```

Comment:
Can we add some metadata via LABEL here? Similar to [these lines](https://github.com/British-Oceanographic-Data-Centre/amrit-repos/blob/main/example-python/Dockerfile#L3) in the Python example.

Reply:
Added